docs/CHANGELOG.md

```diff
@@ -15,6 +15,19 @@ Revise stream clipping to WBD by (1) reducing the buffer to clip streams away fr
 
 <br/><br/>
 
+## v4.4.2.4 - 2023-10-11 - [PR#1007](https://github.com/NOAA-OWP/inundation-mapping/pull/1007)
```

Comment:
Minor formatting suggestions: Make the version number `v4.x.x.x` and move this PR to the top of the CHANGELOG. Add `<br/><br/>` at the end of this PR preceded and followed by a blank line.

Reply:
I went ahead and added the new line under the CHANGELOG.MD entry.